storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operation/put/PreparedPutContentValuesIterable.java

```diff
@@ -120,7 +120,7 @@
          *
          * @param putResolver put resolver
          * @return builder
-         * @see {@link DefaultPutResolver} — easy way to create {@link PutResolver}
+         * @see DefaultPutResolver
```

Comment:
Yes, when you use `@see` -> `@link` placed automatically and if you'll write `@see {@link ...}` link will be broken!